cachi2/core/utils.py

```diff
@@ -48,7 +48,7 @@ def run_cmd(cmd: Sequence[str], params: dict) -> str:
             ),
         )
 
-    response = subprocess.run([executable_path, *args], **params)  # nosec
```

Comment:
Lol, okay, so we always ignored literally ALL bandit rules on our own subprocess run wrapper, good to know :D.